README.enduser.md

```diff
@@ -1,16 +1,16 @@
 # Getting started with the UI
 
 ## Main view
-When starting AAT the first time it shows the **main view**. That's basically a list of sub-views you can open. The first 3 entries are **"Copckpit A, Cockpit B, and Map"** if you have one of these open you can cycle through them with the down arrow. The right arrow switches trough the different pages of the view you are in.
+When starting AAT the first time it shows the **main view**. That's basically a list of sub-views you can open. The first 3 entries are **"Cockpit A, Cockpit B, and Map"** if you have one of these open you can cycle through them with the down arrow. The right arrow switches trough the different pages of the view you are in.
 
 ## Map view (or full screen map)
 In the **map view** the down arrow is in the main bar (tap the top area of the map). For zooming and navigating tap the bottom area of the map. For grid, search, overpass etc. tap  the right area of the map. For the draft editor tap the left side of the map. 
 The track that gets currently recorded is always displayed on the map.
 
 ## Display recorded track
-Now once you have stopped the tracker go back to the **main view** with your phones back button. Then select **Track list** to show a list of all recorded GPX tracks. Every track has a preview image if you tap on this image you'll get a context menu for this GPX-file. _Here you can for example select this track for one of the 4 overlay slots._ You can display any recorded track by taping the text area of the list entry. (Always go back to the list with the phones back button). In the track view there are again different pages (graph, summary and map) and you can cycle trough all tracks with the up and down arrow.
+Now once you have stopped the tracker go back to the **main view** with your phones back button. Then select **Track list** to show a list of all recorded GPX tracks. Every track has a preview image if you tap on this image, you'll get a context menu for this GPX-file. _Here you can for example select this track for one of the 4 overlay slots._ You can display any recorded track by taping the text area of the list entry. (Always go back to the list with the phones back button). In the track view there are again different pages (graph, summary and map) and you can cycle trough all tracks with the up and down arrow.
```

Comment:
It's not clear what changed here @CornyjK - please can you check this.

Reply:
@boite `if you tap on this image, you'll get a context menu` > `if you tap on this image, you'll get a context menu` (comma).